### ISSUE TITLE
Dismount Epona and Carrots Show on-screen randomly

### ISSUE THREAD

**Issue by yulayula16** [TestRunnerSRL/OoT-Randomizer#1001]
I was playing through a freshly generated seed and I encountered a weird bug where I'll be shown Carrots on the screens as if you were on Epona while NOT on Epona and the A button will then show Faster text, not sure how or why this happened, like you could be anywhere like a town or dungeon and it shows up on screen. You can't put away your sword when this bug occurs, but you can still make Navi go away, pick up stuff with A and open doors and chests with A.

Restarting the game on emulator doesn't make this bug go away, you pretty much have to close the emulator entirely and load the game up as normal. I'm using RetroArch with Mupen64Plus core. I've found it to be reproducible as it would come back later, I'm not too entirely sure what triggers the bug, but one thing consistent that made it appear was being on the pause menu for a while and backing out.

managed to get a clip of the bug randomly appearing from a stream.
https://www.twitch.tv/videos/558459608

**Comment by flagrama** [comment]
This is a mupen64plus Next issue, not a rando issue. This is actually an improvement to the random freeze that used to happen.

**Comment by Tantalus-X** [comment]
![OoTR_346432_D8SW1MQ9T4-200212-143853](https://user-images.githubusercontent.com/31192532/75619458-33b82580-5b7c-11ea-857b-a74cd163b2b9.png) 

i have epona gui as a child

**Comment by flagrama** [comment]
Yes. You will have to save, close content, then reload the ROM. This is a emulator issue, not a rando issue. It is also better than the old freezing issue.

**Comment by flagrama** [comment]
We're been working with the emulator developer to work on a fix, but for now the current code workaround of "Show some dumb UI issue instead of hard freezing" is the preferred hotfix.
